features/src/main/scala/com/salesforce/op/features/FeatureJsonHelper.scala

```diff
@@ -114,6 +118,11 @@ object FeatureJsonHelper {
     val isResponse = (json \ "isResponse").extract[Boolean]
     val originStageUid = (json \ "originStage").extract[String]
     val parentUids = (json \ "parents").extract[Array[String]]
+    // TODO get this to work
```

Comment:
what's the problem here?

Reply:
the case class is not available in this module...I can move it to from core to features. though at that point I should probably get rid of the base class...so kind of a cascade of changes.

Reply:
If you can move it without changing the package - do it.

Reply:
I will try it in a separate PR, 'cause I am on a deadline :-P It will not be moving just one file there are at least 3 other classes that would need to move to move FeatureDistribution